cmd/demo-app/main.go

```diff
@@ -54,31 +69,16 @@ func run(ctx context.Context, log *logrus.Logger) {
 		walker.ChanWalkDir(ctx, viper.GetString("dirPath"), log),
 		worker.NewWorker(ctx, viper.GetString("algorithm"), log),
 	)
-	switch {
-	case viper.GetBool("doHelp"):
-		flag.Usage = func() {
-			fmt.Fprintf(os.Stderr, "Custom help %s:\nYou can use the following flag:\n", os.Args[0])
-
-			flag.VisitAll(func(f *flag.Flag) {
-				fmt.Fprintf(os.Stderr, "  flag -%v \n       %v\n", f.Name, f.Usage)
-			})
-		}
-		flag.Usage()
-	case len(viper.GetString("dirPath")) > 0:
-		for {
-			select {
-			case hashData, ok := <-hashesChan:
-				if ok {
-					fmt.Printf("%s %s\n", hashData.Hash, hashData.Path)
-				} else {
-					return
-				}
-			case <-ctx.Done():
-				fmt.Println("program termination after receiving a signal")
+	for {
+		select {
+		case hashData, ok := <-hashesChan:
+			if !ok {
 				return
 			}
+			fmt.Printf("%s %s\n", hashData.Hash, hashData.Path)
```

Comment:
fmt => log

Reply:
this is not log message it is output of cmd util